src/ClientData/include/ClientData/ProcessData.h

```diff
@@ -91,7 +98,7 @@ class ProcessData final {
 
   // This is a map from module_path to the space in memory where that module is loaded
   absl::node_hash_map<std::string, ModuleInMemory> module_memory_map_;
-  std::map<uint64_t, std::string> start_addresses_;
+  std::map<uint64_t, ModuleInMemory> start_addresses_;
```

Comment:
Can we take rename this to `start_address_to_module_in_memory_` or something like that? The name is currently hinting at this being a set.

Reply:
done.